src/gen/api/appStoreApi.ts

```diff
@@ -0,0 +1,139 @@
+/**
+ * Xero AppStore API
+ * These endpoints are for Xero Partners to interact with the App Store Billing platform
+ *
+ * The version of the OpenAPI document: 2.12.0
```

Comment:
should this be 2.15.0?

Reply:
Yeah I will regen once I've got master merged for OAS pr which will update this 

https://github.com/XeroAPI/Xero-OpenAPI/pull/440/files